### DISCUSSION TITLE
⭐️ Starstruck Exchange ⭐️

### DISCUSSION THREAD

**Discussion by orzklv** [Schweinepriester/github-profile-achievements#688] [General]
Star mine and I will do the same.

https://github.com/katsuki-yuri/neko

**Comment by PARKER-X** [comment]
https://github.com/PARKER-X/ConfidentialRoom/

**Reply by PARKER-X** [reply]
> https://github.com/PARKER-X/ConfidentialRoom/

@katsuki-yuri

**Reply by orzklv** [reply]
done!

**Reply by HasanSafarli** [reply]
> https://github.com/PARKER-X/ConfidentialRoom/

starred you, star me too please

https://github.com/HasanSafarli/food-delivery-app

**Reply by Veercodeprog** [reply]
https://github.com/Veercodeprog/full-stack-assignment.ask me when you done starring mine

**Comment by canassa** [comment]
https://github.com/canassa/minibar

**Reply by orzklv** [reply]
done!

**Reply by PARKER-X** [reply]
> https://github.com/canassa/minibar

Done bro
https://github.com/PARKER-X/BarBar

**Reply by gelfik** [reply]
> > https://github.com/canassa/minibar
> 
> Done bro https://github.com/PARKER-X/BarBar

done, go me plz bro
https://github.com/gelfik/gelfik

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by canassa** [reply]
> Star my project and comment below, I will star yours too:
> 
> https://github.com/itspasindu/isipathana-college-demo-site

Done

**Reply by Veercodeprog** [reply]
https://github.com/Veercodeprog/full-stack-assignment.ask me when you done starring mine

**Comment by itspasindu** [comment]
Done.
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by orzklv** [reply]
Done!

**Reply by PARKER-X** [reply]
> Done. please star my repo too: https://github.com/itspasindu/isipathana-college-demo-site

Done bro
https://github.com/PARKER-X/BarBar

**Reply by gelfik** [reply]
done
https://github.com/gelfik/gelfik

**Reply by itspasindu** [reply]
> done https://github.com/gelfik/gelfik

done

**Reply by Veercodeprog** [reply]
https://github.com/Veercodeprog/full-stack-assignment.ask me when you done starring mine

**Comment by HasanSafarli** [comment]
https://github.com/HasanSafarli/food-delivery-app

**Reply by orzklv** [reply]
Done!

**Reply by PARKER-X** [reply]
https://github.com/PARKER-X/BarBar
Done bro

**Reply by itspasindu** [reply]
> https://github.com/PARKER-X/BarBar
> Done bro

Done 👍

**Reply by gelfik** [reply]
done
https://github.com/gelfik/gelfik

**Reply by HasanSafarli** [reply]
done yall

**Reply by Veercodeprog** [reply]
https://github.com/Veercodeprog/full-stack-assignment.ask me when you done starring mine

**Comment by gelfik** [comment]
> Star mine and I will do the same.
> 
> https://github.com/katsuki-yuri/neko

done, go me plz bro
https://github.com/gelfik/gelfik

**Reply by orzklv** [reply]
Done!

**Reply by orzklv** [reply]
hold on, you sure that you starred mate?) https://github.com/katsuki-yuri/neko/stargazers

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by orzklv** [reply]
> Star my project and comment below, I will star yours too:
> 
> [itspasindu/isipathana-college-demo-site](https://github.com/itspasindu/isipathana-college-demo-site)

already starred mate

**Reply by ArielMAJ** [reply]
@gelfik I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Veercodeprog** [reply]
https://github.com/Veercodeprog/full-stack-assignment.ask me when you done starring mine

**Comment by ArielMAJ** [comment]
@katsuki-yuri I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by orzklv** [reply]
Done!

**Reply by ArielMAJ** [reply]
Ty!

**Comment by Veercodeprog** [comment]
You can take part in ongoing [coding problem](https://github.com/Veercodeprog/Data-Structures-implementation/discussions/categories/coding-discussion-share-your-solution-with-best-time-complexity-approach) discussion.

A daily problem will be posted, and the answer with the most efficient time complexity submitted first will be selected as the accepted solution.

**Reply by ArielMAJ** [reply]
Nice

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment

**Reply by ArielMAJ** [reply]
@Veercodeprog I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by orzklv** [reply]
Done!

**Reply by Veercodeprog** [reply]
nice blog website

**Reply by orzklv** [reply]
Thanks :D

**Comment by hassoonsy2** [comment]
https://github.com/hassoonsy2/hassoonsy2

**Reply by orzklv** [reply]
mason, where is the star?

**Reply by ArielMAJ** [reply]
@hassoonsy2 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by hassoonsy2** [reply]
@katsuki-yuri i already did it :)

**Reply by hassoonsy2** [reply]
@ArielMAJ done 🫡

**Reply by orzklv** [reply]
@hassoonsy2 done comrade :)

**Reply by ArielMAJ** [reply]
Ty!

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰
